install/recorders.csv

```diff
@@ -1261,7 +1261,7 @@ Kinemetrics,QDR,QDR,0189,904F,20,270,0,0,2000-08-04T00:00:00Z,2014-07-09T23:00:0
 Kinemetrics,QDR,QDR,0190,924H,20,218,0,0,2000-07-07T00:00:00Z,2014-07-15T00:00:00Z
 Kinemetrics,QDR,QDR,0191,904E,20,270,0,0,2000-08-04T00:00:00Z,2014-07-09T23:00:00Z
 Kinemetrics,QDR,QDR,0192,924J,20,128,0,0,2000-07-07T00:00:00Z,2014-07-15T00:00:00Z
-Nanometrics,Titan,Titan,0390,WEL,22,0,0,0,2016-05-06T00:05:00Z,2016-08-29T03:00:00Z
+Nanometrics,Titan SMA,Titan SMA,0390,WEL,22,0,0,0,2016-05-06T00:05:00Z,2016-08-29T03:00:00Z
```

Comment:
Could we flip this back to a "Titan" for the sensor, and a "Titan SMA" for the datalogger (impacts the response building).

Reply:
Done

Reply:
We can use it as a template for the next deploys, and if we decide we don't have any data for the evaluation box, we can remove it then.